File: harvey/1924.py
m, d = map(int, input().split())

for i in range(1, m):
    if i in [1,3,5,7,8,10,12]:
        d+=31
    elif i in [4,6,9,11]:
        d+=30
    elif i == 2:
        d+=28
day = ['SUN','MON','TUE','WED','THU','FRI','SAT']
print(day[d%7])

File: chu/solved/1924.js
(function main(){
    var year = [
        [1, 31],
        [2, 28],
        [3, 31],
        [4, 30],
        [5, 31],
        [6, 30],
        [7, 31],
        [8, 31],
        [9, 30],
        [10, 31],
        [11, 30],
        [12, 31]
    ];
    var day = [ 'SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT' ];

    var n = require('fs').readFileSync('../stdin/1924').toString().split(' ');
    var sum = 0;
    for(var i = 1; i < parseInt(n[0]); i++){
        sum += year[i-1][1];
    }
    sum = sum + parseInt(n[1]);
    
    console.log(sum);
    sum = sum % 7;
    console.log(sum);
    console.log(day[sum]);
})()